### PR TITLE
Transactional Messaging 

### PR DESCRIPTION
**Pull request by nickrobin**
# Pull Request/Issue Resolution

**Description of Change:**
updating API documentation to include new transactional messaging endpoint that will be available for beta soon to select customers. 

### Is this change associated with a Braze feature/product release?
- [X] Yes (__TBD - do not merge yet__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

**Comment by KellieHawks** [comment]
Hey @nickrobin, fixed that table and made a couple of grammar/typo edits.

**Comment by nickrobin** [comment]
hey @KellieHawks when you get a second, I'm trying to add the page I created for transactional messaging campaign creation flow as a hidden page on a route like https://www.braze.com/docs/api/api_campaigns/transactional_campaigns but didn't know the best way to go about that so looking for any guidance you might have as to create the page that way. If it's a pain, I'm happy to put it into the hidden folder as well

**Comment by KellieHawks** [comment]
Hey @nickrobin , 
Added that article as: www.braze.docm/docs/api/api_campaigns/transactional_campaigns by using an alias link.
Also went through and did a couple of grammar edits and image size edits. 👍🏻

**Comment by nickrobin** [comment]
@mattdisipio draft of docs here for your review, you can view the original files or check the staging deployment at the links below:

https://braze-docs-staging-pr-1400.herokuapp.com/docs/api/transactional_message_api_campaign/
https://braze-docs-staging-pr-1400.herokuapp.com/docs/api/endpoints/messaging/send_messages/post_send_transactional_message/

**Comment by mattdisipio** [comment]
thanks, we're going to need to update the endpoint spec once MAPI-42 is done but will review the rest

File: _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+platform: API
+tool:
+  - Campaigns
+hidden: true
+description: "This article outlines details about the Send Transactional Message via API Triggered Delivery Braze endpoint."
+---
+{% api %}
+# Sending Transactional Messages via API Triggered Delivery
+{% apimethod post %}
+/v2/transactional/campaigns/YOUR_CAMPAIGN_ID_HERE/send/
```
Comment (mattdisipio) [inline _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md:15]: needs to change to `/transactional/v1/campaigns/YOUR_CAMPAIGN_ID_HERE/send`

File: _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  -d '{
+        "trigger_properties": {
+          "example_string_property": YOUR_EXAMPLE_STRING,
+          "example_integer_property": YOUR_EXAMPLE_INTEGER
+        },
+        "recipient": {
+          "external_user_id": TARGETED_USER_ID_STRING
+        }
+      }' \
+  https://rest.iad-01.braze.com/v2/transactional/campaigns/YOUR_TRANSACTIONAL_CAMPAIGN_ID/send
```
Comment (mattdisipio) [inline _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md:80]: needs to change to `/transactional/v1/campaigns/YOUR_CAMPAIGN_ID_HERE/send`

Comment (nickrobin) [reply _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md]: updated

File: _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md (hunk 1 above)
Comment (nickrobin) [reply _docs/_api/endpoints/messaging/send_messages/post_send_transactional_message.md]: updated

**Pull request merged**